### PR TITLE
Splitted the funciton getEndpoints for a more readable code

### PR DESCRIPTION
**Pull request by maziland**
A small change, for a better and more readable code

### CODE REVIEW

File: controllers/endpoints.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ function formatDate(dateString) {
     return `${month}/${day}/${year}, ${hours}:${minutes}:${seconds}`;
 }
 
-async function getEndpoints(req, res) {
+async function getSingleEndpoint(req, res) {
```
Comment (tal-az) [inline controllers/endpoints.js:17]: Just written my own versions using an API call for single host by hostID, check it out it might match your needs.

Comment (tal-az) [reply controllers/endpoints.js:17]: in my last PR for rmNode

**Review by tal-az — CHANGES_REQUESTED** [review]

File: controllers/endpoints.js (hunk 1 above)
Comment (maziland) [reply controllers/endpoints.js:17]: I think you got confused.
This is a controller function to return an EJS file for single endpoint only, now fetching 1 endpoint data from fleet

Comment (maziland) [reply controllers/endpoints.js:17]: Ok i understand what you suggested. But i can't use it before you merge it to main :)

Comment (tal-az) [reply controllers/endpoints.js:17]: Go on and merge then, we'll change it in the future don't let it stop the feature

**Review by tal-az — APPROVED** [review]

**Pull request merged**